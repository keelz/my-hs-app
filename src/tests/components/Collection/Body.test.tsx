import * as React from 'react';
import * as enzyme from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import CollectionBody from '../../../components/Collection/Body';

const defaultProps = () => Object.freeze({
  collection: {
    cards: Array(0),
  },
});

describe('Collection Body', () => {
  describe('snapshots', () => {
    it('renders without crashing', () => {
      const tree = enzyme.shallow(<CollectionBody {...defaultProps()} />);
      expect(shallowToJson(tree)).toMatchSnapshot();
    });
  });
});
