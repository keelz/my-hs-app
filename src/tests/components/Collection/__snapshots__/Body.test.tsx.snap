// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Collection Body snapshots renders without crashing 1`] = `
<div
  className="Collection-body"
>
  <BodyNavButton
    align={3}
  />
  <BodyNavButton
    align={1}
  />
</div>
`;
